slacker/__init__.py

```diff
@@ -96,7 +96,10 @@ def _request(self, method, api, **kwargs):
 
         response = Response(response.text)
         if not response.successful:
-            raise Error(response.error)
+            error = response.error
+            if response.message:
+                error = "{error}: {response.message}"
+            raise Error(error)
```

Comment:
Could you remove this section as it's not directly related to this PR? Although I'm not a big fan of this as I want to keep the implementation as close as the original API as possible.